swarm/storage/localstore.go

```diff
@@ -194,20 +194,25 @@ func (ls *LocalStore) Close() {
 	ls.DbStore.Close()
 }
 
-// Migrate checks the datastore schema vs the runtime schema, and runs migrations if they don't match
+// Migrate checks the datastore schema vs the runtime schema and runs
+// migrations if they don't match
 func (ls *LocalStore) Migrate() error {
 	actualDbSchema, err := ls.DbStore.GetSchema()
 	if err != nil {
 		log.Error(err.Error())
 		return err
 	}
 
-	log.Debug("running migrations for", "schema", actualDbSchema, "runtime-schema", CurrentDbSchema)
-
 	if actualDbSchema == CurrentDbSchema {
 		return nil
 	}
 
+	log.Debug(
```

Comment:
why on separate lines? I haven't seen this before in the codebase

Reply:
👍

Reply:
- The line was longer than 80 char => breaks
- I did see it somewhere this way, that is why I did not break every arg to a separate line as I usually do.

dropping change